scipy/constants/__init__.py

```diff
@@ -5,316 +5,118 @@
 
 .. currentmodule:: scipy.constants
 
-Physical and mathematical constants and units.
 
+About the SI units
```

Comment:
This is reference documentation, so background material would better go to the bottom of the page, as people are mainly going to be looking for variable/function names etc. (except maybe the first time when reading this).

Reply:
Okay, I'll do it later.